docker-compose.ci.yml

```diff
@@ -18,6 +18,23 @@ x-web:
   expose:
     - 8000
 
+# Define common behavior of worker
+x-worker:
+  &default-worker
+  image: erica_worker:latest
+  build:
+    context: .
+    target: worker
+    args:
+      bucket_name: ${ERICA_BUCKET_NAME}
+      access_key_id: ${ACCESS_KEY_ID}
+      access_key: ${ACCESS_KEY}
+      endpoint_url: ${ENDPOINT_URL}
+      elster_datenlieferant: ${ELSTER_DATENLIEFERANT}
+      elster_hersteller_id: ${ELSTER_HERSTELLER_ID}
+  expose:
+    - 8000
```

Comment:
Is this right? we should not use 8000 or? The worker and the API should use different ports or?

Reply:
We don't even have to expose the port for the workers, do we?

Reply:
The image is called erica_worker but it does not only contain the worker but also the API. This is the image to be used in the VM, which has both the API and the worker definition. On the VM this image is the only one where the property DONGLE_CONNECTED is set to True, meaning the API v1 is only running on the VM.
So we do need to expose the port, is not used in the worker but in the API